contribs/taxi/src/main/java/org/matsim/contrib/taxi/run/MultiModeTaxiModule.java

```diff
@@ -36,8 +39,9 @@ public class MultiModeTaxiModule extends AbstractModule {
 	@Override
 	public void install() {
 		for (TaxiConfigGroup taxiCfg : multiModeTaxiCfg.getModalElements()) {
-			install(new TaxiModeModule(taxiCfg));
-			installQSimModule(new TaxiModeQSimModule(taxiCfg));
+			var drtCfg = TaxiAsDrtConfigGroup.convertTaxiToDrtCfg(taxiCfg);
+			install(new DrtModeModule(drtCfg));
+			installQSimModule(new DrtModeQSimModule(drtCfg, new TaxiModeOptimizerQSimModule(taxiCfg)));
```

Comment:
This is the core change: we switch to `DrtModeModule` and `DrtModeQSimModule`. To make it work, we need to convert the taxi config to a drt config that is passed to those modules.